app.rb

```diff
@@ -0,0 +1,30 @@
+require_relative 'database'
+require 'sinatra/base'
+require './lib/peep.rb'
+
+
+class Chitter < Sinatra::Base
+  enable :sessions
+

```

Comment:
Apparently it's good practice to have multiple controller files (i.e. one for Peeps, one for the general application)